AsyncDisplayKit/ASCellNode.m

```diff
@@ -36,7 +38,9 @@ - (instancetype)init
   // Use UITableViewCell defaults
   _selectionStyle = UITableViewCellSelectionStyleDefault;
   self.clipsToBounds = YES;
-
+  if ([self.layoutDelegate respondsToSelector:@selector(visibleNodeDidScroll:inScrollView:withCellFrame:)]) {
```

Comment:
Layout delegate will not be set here.  This point, we should only check [self respondsToSelector:(the selector that subclasses of cell node can override)]